lib/bios/menus.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
```

Comment:
It's a third PR where I see this file being added. It makes the review process to be confusing.